cmd/tx_lookup.cpp

```diff
@@ -26,6 +26,13 @@
 
 using namespace silkworm;
 
+static Bytes compact(Bytes &b) {
+    for(size_t current_index = 0; current_index < b.size(); current_index++) {
+        if (b[current_index] != 0) return b.substr(current_index);
+    }
+    return Bytes(1, '\0');
+}
```

Comment:
I think this might have a problem if no leading zeroes are found: in such case you're returning a string of 1 char set to zero. (talking block height as is now is unlikely we fill a uint64 completely but ... the logic is wrong).
I would suggest to go for a more concise way as :

```
static Bytes compact(Bytes &b) {
    std::string::size_type offset{b.find_first_not_of((uint8_t)0)};
    if (offset != std::string::npos) {
        return b.substr(offset);
    }
    return b;
}
```